README.md

```diff
@@ -35,23 +35,27 @@ You can also chat with us in [Slack](https://homebridge-slackin.glitch.me).
 
 ## Installation
 
+The [Homebridge Wiki](https://github.com/nfarina/homebridge/wiki) contains step-by-step instruction on how to install Node.js and setup Homebridge as a service so it automatically start on boot:
+
+* [Setup Homebridge on a Raspberry Pi (Raspbian)](https://github.com/nfarina/homebridge/wiki/Install-Homebridge-on-Raspbian)
+* [Setup Homebridge on Debian/Ubuntu Linux](https://github.com/nfarina/homebridge/wiki/Install-Homebridge-on-Debian-or-Ubuntu-Linux)
+* [Setup Homebridge on macOS](https://github.com/nfarina/homebridge/wiki/Install-Homebridge-on-macOS)
+* [Setup Homebridge on Windows 10](https://github.com/nfarina/homebridge/wiki/Install-Homebridge-on-Windows-10)
+* [Setup Homebridge on Docker (Linux)](https://github.com/nfarina/homebridge/wiki/Install-Homebridge-on-Docker)
+* [Other Platforms](https://github.com/nfarina/homebridge/wiki/Other-Platforms)
+
 #### Quick Overview
-1. **Node v4.3.2 or greater is required.** Check by running: `node --version`. The plugins you use may require newer versions.
-2. **On Linux only:** Install the libavahi-compat-libdnssd-dev package: `sudo apt-get install libavahi-compat-libdnssd-dev`
```

Comment:
`libavahi-compat-libdnssd-dev` is no longer required since moving to pure javascript based bonjour (https://github.com/KhaosT/HAP-NodeJS/pull/495).